src/contracts/StablecoinConverter.d.ts

```diff
@@ -0,0 +1,188 @@
+/* Generated by ts-generator ver. 0.0.8 */
+/* tslint:disable */
+
+import BN from 'bn.js'
+import { Contract, ContractOptions } from 'web3-eth-contract'
+import { EventLog, Callback, EventEmitter } from 'web3-core'
+import { TransactionObject, BlockType } from 'web3/eth/types'
+import { ContractEvent } from './types'
+
+interface EventOptions {
+  filter?: object
+  fromBlock?: BlockType
+  topics?: string[]
+}
+
+export class StablecoinConverter extends Contract {
```

Comment:
Is there a way we can share this type definition between `dex-react` and this repo?
I'm afraid we'll soon have divergences and double maintenance pain.

Reply:
Sure, I think at some point we should create a lib for the smart contract. I was waiting until we figure out what we the typescript in both projects.  Ideally it should be autogenerated and published in a NPM package that we can use ourselves and offer two 3rd parties

This file u are referring to, I autogenerated it using a tool, but this tool is a bit flaky right now, so we still need to find an alternative.

Reply:
Do we have a task in the backlog already for this?

Reply:
Now we do. That would be a first draft of it: https://github.com/gnosis/dex-react/issues/200